internal/common/cloudproviderapi/client.go

```diff
@@ -117,11 +118,12 @@ type AWSCloudWatchScrapeJobResponse struct {
 	Services             []AWSCloudWatchService         `json:"services"`
 	CustomNamespaces     []AWSCloudWatchCustomNamespace `json:"customNamespaces"`
 	// computed fields beyond the original request
-	RoleARN                   string   `json:"roleARN"`
-	Regions                   []string `json:"regions"`
-	RegionsSubsetOverrideUsed bool     `json:"regionsSubsetOverrideUsed"`
-	DisabledReason            string   `json:"disabledReason"`
-	Provenance                string   `json:"provenance"`
+	RoleARN                   string            `json:"roleARN"`
+	Regions                   []string          `json:"regions"`
+	RegionsSubsetOverrideUsed bool              `json:"regionsSubsetOverrideUsed"`
+	DisabledReason            string            `json:"disabledReason"`
+	Provenance                string            `json:"provenance"`
+	StaticLabels              map[string]string `json:"staticLabels"`
```

Comment:
nit: this should be moved to below the `CustomNamespaces` field, since it's not a "computed field beyond the original request", but instead is a field that is part of the original request